psd-web/spec/models/user_spec.rb

```diff
@@ -1,21 +1,87 @@
 require "rails_helper"
 
-RSpec.describe User, with_keycloak_config: true do
-  before do
-    allow(ENV).to receive(:fetch).with("KEYCLOAK_AUTH_URL").and_return("test")
-    allow(ENV).to receive(:fetch).with("KEYCLOAK_CLIENT_ID").and_return(client_id)
-    allow(ENV).to receive(:fetch).with("KEYCLOAK_CLIENT_SECRET").and_return(client_secret)
-    allow(KeycloakToken).to receive(:new).and_return(token_stub)
+RSpec.describe User do
+  describe ".activated" do
+    before do
+      @unactivated_user = create(:user)
+      @activated_user = create(:user, :activated)
+    end
+
+    it "returns only users with activated accounts" do
+      expect(User.activated.to_a).to eq [@activated_user]
+    end
+  end
+
+  describe "#activate!" do
+    subject(:user) { build(:user) }
+
+    it "sets the activated flag on the user" do
+      expect(user.account_activated?).to be_falsy
```

Comment:
why check for falsey instead of false? also there is nicer way to right these tests.
```suggestion
      expect(user).not_to be_account_activated
```

Reply:
This was redundant anyway after I added a trait to the factory so we can be explicit in the state of the user at the time it is created.